crates/pallets/starknet/src/lib.rs

```diff
@@ -718,7 +722,8 @@ pub mod pallet {
             let pending = Self::pending();
 
             let global_state_root = U256::zero();
-            let sequencer_address = ContractAddressWrapper::default();
+            let sequencer_address =
+                [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2];
```

Comment:
explain why this hardcoded value in a comment. explain and create an issue on how to remove the hardcoded value